Jenkinsfile

```diff
@@ -91,7 +94,7 @@ pipeline {
         stage('Azure Authenticator') {
           steps {
             script {
-              while (!env.AZURE_AUTHN_INSTANCE_IP?.trim()) {
+              while (!env.AZURE_AUTHN_INSTANCE_IP?.trim() || !env.SYSTEM_ASSIGNED_IDENTITY?.trim()) {
```

Comment:
why we need !env.SYSTEM_ASSIGNED_IDENTITY?.trim() here?

Reply:
these 2 variables are required for authn-azure tests and are retrieved from the parallel step where we allocate the azure vm. for this reason we wait for the tests to run until we have those variables in the env